python/cugraph-pyg/cugraph_pyg/tests/test_cugraph_loader.py

```diff
@@ -40,7 +43,9 @@
 
 
 @pytest.mark.skipif(isinstance(torch, MissingModule), reason="torch not available")
-def test_cugraph_loader_basic(karate_gnn):
+def test_cugraph_loader_basic(
+    karate_gnn: Tuple[FeatureStore, Dict[str, np.ndarray], Dict[str, int]]
```

Comment:
The type of G (the 2nd return value) should be `Dict[Tuple[str, str, str], ......] ` to be exact. I think the CuGraphStore class has the same typing issue.

Reply:
fixed